packages/core/src/Stores/client-store.js

```diff
@@ -1501,6 +1500,11 @@ export default class ClientStore extends BaseStore {
     }
 
     async resetVirtualBalance() {
+        Analytics.trackEvent('ce_tradershub_dashboard_form', {
+            action: 'reset_balance',
+            form_name: 'traders_hub_default',
+            account_mode: { ...this.accounts[this.loginid] }.is_virtual === 1 ? 'demo' : 'real',
```

Comment:
not sure but i think account will be always demo here since we cannot reset virtual balance when we are in real account